rpc/gnmi/gnmi-specification.md

```diff
@@ -793,9 +776,9 @@ Stream subscriptions are long-lived subscriptions which continue to transmit upd
 
 A `STREAM` subscription is created by sending a `SubscribeRequest` message with the subscribe field containing a `SubscriptionList` message with the type specified as `STREAM`. Each entry within the `Subscription` message is specified with one of the following `modes`:
 
-
-
-*   **On Change (`ON_CHANGE`)** - when a subscription is defined to be "on change", data updates are only sent when the value of the data item changes. A heartbeat interval MAY be specified along with an "on change" subscription - in this case, the value of the data item(s) MUST be re-sent once per heartbeat interval regardless of whether the value has changed or not.
+*   **On Change (`ON_CHANGE`)** - when a subscription is defined to be "on change", data updates are only sent when the value of the data item changes.
+    * For all `ON_CHANGE` subscriptions, the target MUST generate updates for all path that match the subscription path(s), and transmit them. Following this initial set of updates, updated values SHOULD only be transmitted when their value changes.
```

Comment:
... for all *paths* that match ...

maybe clarify that the target must *first* generate updates for all paths as with any other subscription